docs_source_files/content/webdriver/driver_requirements.es.md

```diff
@@ -4,26 +4,40 @@ weight: 2
 ---
 
 
-A través del Webdriver, Selenium es capaz de soportar los navegadores mas usados en el mercado como Chrom(ium), Firefox, Internet Explorer, Opera y Safari.
-Webdriver maneja los navegadores, cuando es posible,  apoyándose en las propias funciones que el navegador incorpora para la automatización.
+A través del WebDriver, Selenium es capaz de soportar los navegadores mas usados 
+en el mercado como Chrom(ium), Firefox, Internet Explorer, Opera y Safari.
```

Comment:
```suggestion
en el mercado como Chrome/Chromium, Firefox, Internet Explorer, Opera y Safari.
```

Reply:
`Chrom(ium)` is used in all other texts, so let's leave it like that. Also `Chrom(ium)`, also refers to the new Edge.

Reply:
Thanks for the info @diemol, I suggested the change because the other docs have text like this. Chrome/Chromium. I will cover modify those in regular PR.